client/client.go

```diff
@@ -429,6 +429,7 @@ type SysInfo struct {
 
 	Refresh     RefreshInfo `json:"refresh,omitempty"`
 	Confinement string      `json:"confinement"`
+	BuildID     string      `json:"build-id"`
```

Comment:
the placement is a bit odd here, it seems it would belong together after version